content/en/docs/concepts/services-networking/service.md

```diff
@@ -14,24 +14,25 @@ weight: 10
 
 <!-- overview -->
 
-{{< glossary_definition term_id="service" length="short" >}}
+{{< glossary_definition term_id="service" length="short" prepend="In Kubernetes, a Service is" >}}
 
-With Kubernetes you don't need to modify your application to use an unfamiliar service discovery mechanism.
-Kubernetes gives Pods their own IP addresses and a single DNS name for a set of Pods,
-and can load-balance across them.
+A key aim of Services in Kubernetes is that you don't need to modify your existing
+application to use an unfamiliar service discovery mechanism.
+You can run code in Pods, whether this is a code designed for a cloud-native world, or
+an older app you've containerized. You use a Service to make that set of Pods available
+on the network so that clients can interact with it.
 
-<!-- body -->
-
-## Motivation
+If you use a {{< glossary_tooltip term_id="deployment" >}} to run your app,
+that Deployment can create and destroy Pods dynamically, as needed - whether that
+is to handle failure, or to match against demand (horizontal scaling).
 
 Kubernetes {{< glossary_tooltip term_id="pod" text="Pods" >}} are created and destroyed
-to match the desired state of your cluster. Pods are nonpermanent resources.
-If you use a {{< glossary_tooltip term_id="deployment" >}} to run your app,
-it can create and destroy Pods dynamically.
+to match the desired state of your cluster. Pods are emphemeral resources (you should not
+expect that an individual Pod is reliable and durable).
 
-Each Pod gets its own IP address, however in a Deployment, the set of Pods
-running in one moment in time could be different from
-the set of Pods running that application a moment later.
+Each Pod gets its own IP address - Kubernetes networking guarantees that.
```

Comment:
hehe, I would not claim Kubernetes guarantees that, it is possible to have duplicated IPs on PodIP since that depend son the CNI plugin cc @thockin

Reply:
:grimacing: fair enough - I'll reword to explain the _intended_ outcome.